scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
 		font-size: 16px;
 	}
-	@media ( max-width: 500px ) {
-		font-size: 14px;
+	@media ( min-width: $viewport-sm ) {
+		font-size: 20px;
```

Comment:
Yup. Typography was created before jsass became a thing we were using, we need to go through Typography and change a few things to variables.